index.bs

```diff
@@ -787,6 +796,19 @@ option. If {{UnderlyingSource/type}} is set to undefined (including via omission
      |underlyingSource|, |underlyingSourceDict|, |highWaterMark|, |sizeAlgorithm|).
 </div>
 
+<div algorithm>
+ The static <dfn id="rs-from" method for="ReadableStream">from(|asyncIterable|)</dfn> method steps
+ are:
+
+ 1. Return ? [$ReadableStreamFromIterable$](|asyncIterable|).
+</div>
+
+<div algorithm>
+ The static <dfn id="rs-of" method for="ReadableStream">of(...|chunks|)</dfn> method steps are:
+
+ 1. Return ? [$ReadableStreamFromIterable$](|chunks|).
```

Comment:
This doesn't really work, as |chunks| is an Infra list of `any` values, not a JS array. A separate abstract op is probably best.

But personally, I'm not sure it's worth adding `of()` vs. just `from([])`. I rarely see people use `Array.of()`.

Reply:
(Oh wow, that was a fast review. 😅)

Hmm, fair point. I believe the argument behind `of()` was because people might accidentally use `ReadableStream.from(uint8Array)` and get a stream of individual bytes instead of a stream with one byte chunk ([comment](https://github.com/whatwg/streams/issues/1018#issuecomment-587350179)). But then again, that's easily fixed with `ReadableStream.from([uint8Array])`.

I'll remove `of()` for now.